ccd-definition/ComplexTypes/DefaultJudgmentSDO/TrialHearingTime-DJ-SDO-HNL-nonprod.json

```diff
@@ -0,0 +1,54 @@
+[
+  {
+    "ID": "TrialHearingTimeDJ",
+    "ListElementCode": "date1",
+    "FieldType": "Date",
+    "ElementLabel": "Date from",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "TrialHearingTimeDJ",
+    "ListElementCode": "date2",
+    "FieldType": "Date",
+    "ElementLabel": "Date to",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "TrialHearingTimeDJ",
+    "ListElementCode": "hearingTimeEstimate",
+    "FieldType": "FixedRadioList",
+    "FieldTypeParameter": "TrialHearingTimeEstimate",
+    "ElementLabel": "The time estimate is",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "TrialHearingTimeDJ",
+    "ListElementCode": "otherHours",
+    "FieldType": "Number",
+    "ElementLabel": "Hours",
+    "SecurityClassification": "Public",
+    "RegularExpression": "^(,[0-9]|[1-9][0-9]|[1-9][0-9][0-9])$"
```

Comment:
there is a leading ',' in the regex which shouldn't be there
(,[

Reply:
This has been amended